packages/fxa-content-server/tests/functional.js

```diff
@@ -81,13 +82,13 @@ module.exports = testsSettingsV2.concat([
   'tests/functional/sync_v3_settings.js',
   'tests/functional/tos.js',
   'tests/functional/verification_reminders.js',
+*/
 ]);
 
 // Mocha tests are only exposed during local dev, not on prod-like
 // instances such as latest, stable, stage, and prod. To avoid
 // Teamcity failing trying to run mocha tests, expose an environment
 // variable it can use to skip the mocha tests.
 if (!process.env.SKIP_MOCHA) {
-  module.exports.unshift('tests/functional/mocha.js');
+  //  module.exports.unshift('tests/functional/mocha.js');
```

Comment:
what's up with the mocha tests?

Reply:
ah, actually I have a commit in a branch that fixes them. Vijay caught one little bug, the others are just some navigate/navigateAway replacements. I'll go ahead and add that commit to this branch